src/views/PlanTargets.vue

```diff
@@ -0,0 +1,201 @@
+  },
+  created: function() {
+    const url = "https://api.photonranch.org/api/all/config"
+    axios.get(url).then(response => {
+      for (let site in response.data) {
+        Vue.set(this.site_info, site, {
+          latitude: response.data[site].latitude,
+          longitude: response.data[site].longitude,
+          name: response.data[site].name,
+          siteoffset: response.data[site].TZ_database_name
+        })
+      }
+      Vue.set(this.user, 'localtime', this.site_info.mrc.siteoffset)
+      Vue.set(this.user, 'lat1', this.site_info.mrc.latitude)
+      Vue.set(this.user, 'lon1', this.site_info.mrc.longitude)
+      console.log(error)
+    })
+    .catch(error => {
+      console.log(error)
+    })
+  },
+  methods: {
+    setLatLong() {
+      const selectedOption = document.getElementById('localtime').options[document.getElementById('localtime').selectedIndex];
+      this.user.lat1 = selectedOption.getAttribute('lat');
+      this.user.lon1 = selectedOption.getAttribute('lon');
+    },
+    submitForm() {
+      this.user.offset = new Date(this.user.dateobs).getTimezoneOffset();
+      this.user.localoffset = moment.utc(this.user.dateobs).tz(this.user.localtime).utcOffset()
+
+      var diclist = [];
+
+      if (this.user.tzinfo == 'my') {
+        var starttime=this.user.dateobs
+      } else if (this.user.tzinfo == 'utc') {
+        var starttime=moment(this.user.dateobs).subtract(this.user.offset, 'm').toDate()
+      } else if (this.user.tzinfo == 'lcl') {
+        var starttime=moment(this.user.dateobs).subtract(this.user.offset+this.user.localoffset, 'm').toDate()
+      }
+
+      var endtime = moment(starttime).add(30, 'm').toDate();
+      for (var i = 0; i < this.easylist.length; ++i) {
+        var altstart = helpers.eq2altazWithDate(this.easylist[i].ra, this.easylist[i].dec, this.user.lat1, this.user.lon1, starttime)[0]
+        var altend = helpers.eq2altazWithDate(this.easylist[i].ra, this.easylist[i].dec, this.user.lat1, this.user.lon1, endtime)[0]
+        if (altstart>35 && altend>35) {
+          diclist.push({
+            "name": this.easylist[i].name, 
+            "type": this.easylist[i].group, 
+            "image": "/targs/DefaultTargetImages/"+this.easylist[i].name.replace(/ /g, "")+".jpg",
+            "ra": this.easylist[i].ra,
+            "dec": this.easylist[i].dec,
+            "starttime": starttime,
+            "altstart": altstart,
+            "altend": altend
+            })
+        }
+      }
+      console.log(diclist)
+      this.targlist=diclist;
+
+      if (window.screen.availWidth<970) {
+        setTimeout(function(){ document.getElementById("results").scrollIntoView({behavior: 'smooth'}); }, 30);}
+    },
+  },
+};
+</script>
+
+<style lang="css" scoped>
+  .target-columns {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content:center;
+  }
+  .the-form {
+    max-width: 326px;
+    margin-left: 8%;
+    margin-bottom: 4%;
+  }
+  .the-page{
+    margin-top:2%;
```

Comment:
use padding instead so the page content isn't pushed below the bottom of the screen.